Src/Plugins/Security/Utilities/AlibabaEcsRequestSigner.cs

```diff
@@ -39,6 +39,9 @@ public AlibabaEcsRequestSigner(DateTime? dateTime = null, string signatureNonce
         /// Sign the HTTP request using Alibaba Cloud access key and secret so it
         /// can be used to access Alibaba Cloud service REST API.
         /// </summary>
+        /// <param name="request">Unsigned httpRequest passed in that will be signed for Alibaba Cloud service REST API.</param>
+        /// <param name="accessKey">Alibbaba Cloud access key.</param>
```

Comment:
>Alibbaba [](http://example.com/codeflow?start=36&length=8)

this is misspelled.

how much of an investment are you making in improving the doc comments? If we aren't going to put them in a sufficient level of polish, we shouldn't bother.

Also, weren't you going to broadcast the engineering standards to the group before committing to many fixes? i.e., was there a discussion with the team about our position on this code standard? If we don't like it, we can just disable the check. Please do schedule this conversation, probably invite the whole team and you can review individual checks with us.

Reply:
Fixed the embarrassingly bad misspell of Alibaba (oddly I chose to only misspell in one location - did the key get stuck down ??) :)

Setup a meeting to discuss coding standards in detail.